api/proto/node/node.go

```diff
@@ -116,7 +116,8 @@ func DeserializeBlockchainSyncMessage(d []byte) (*BlockchainSyncMessage, error)
 	decoder := gob.NewDecoder(bytes.NewReader(d))
 	err := decoder.Decode(&blockchainSyncMessage)
 	if err != nil {
-		log.Panic(err)
+		utils.GetLogger().Crit("Error", err)
+		panic(err)
```

Comment:
is log.Panic will panic the process?  Or do we need to panic it here?  @mikedoan please review.

Reply:
[GoDoc](https://golang.org/pkg/log/#Panic)
> Panic is equivalent to Print() followed by a call to panic().

Reply:
@mikedoan , do we need to panic here? please comment.

Reply:
Don't panic here please. We should avoid panic (exit the software) as much as possible